src/chrome/content/rules/Media-Storehouse.xml

```diff
@@ -1,34 +1,32 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://mediastorehouse.com/ => http://mediastorehouse.com/: Redirect for 'http://mediastorehouse.com/' missing Location
-Fetch error: http://mediastorehouse.net/ => https://secure.mediastorehouse.com/: Redirect for 'http://mediastorehouse.net/' missing Location
-Fetch error: http://www.mediastorehouse.net/ => https://secure.mediastorehouse.com/: Redirect for 'http://www.mediastorehouse.net/' missing Location
-	s3.amazonaws.com/images.mediastorehouse.com/
+	Non-functional hosts in *.mediastorehouse.com
+		SSL peer certificate was not OK:
+		- mediastorehouse.com
 
+	Non-functional hosts in *.mediastorehouse.net
+		Timeout was reached:
+		- mediastorehouse.net
 
-	Nonfunctional:
-
-		- images.mediastorehouse.net	(times out; not on secure nor images...com)
-
+		SSL peer certificate was not OK:
+		- images.mediastorehouse.net
+		- www.mediastorehouse.net
 -->
-<ruleset name="Media Storehouse (partial)">
```

Comment:
There are still a lot of uncovered subdomains. Maybe you just want to keep the `partial`?